### PR TITLE
Add information on PHP configuration

### PR DESCRIPTION
**Pull request by jaylinski**
The PHP image will ship with the default development and production
INI files in order to make it easier to securely configure PHP.

See https://github.com/docker-library/php/pull/711 and https://github.com/docker-library/php/issues/692.

### CODE REVIEW

File: php/content.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ PHP is a server-side scripting language designed for web development, but which
 
 %%LOGO%%
 
-# How to use this image.
```
Comment (jaylinski) [inline php/content.md]: Aligns headline with other README (which don't use a dot at the end).

See https://github.com/docker-library/docs/tree/master/wordpress#how-to-use-this-image.

File: php/content.md [hunk 2]
```diff
@@ -56,15 +56,7 @@ $ docker build -t my-php-app .
...
-We recommend that you add a custom `php.ini` configuration. `COPY` it into `/usr/local/etc/php` by adding one more line to the Dockerfile above and running the same commands to build and run:
-
-```dockerfile
-FROM %%IMAGE%%:7.2-apache
-COPY config/php.ini /usr/local/etc/php/
-COPY src/ /var/www/html/
-```
-
-Where `src/` is the directory containing all your PHP code and `config/` contains your `php.ini` file.
+We recommend that you add a `php.ini` configuration file, see [Configuration](#configuration) for details.
```
Comment (jaylinski) [inline php/content.md]: Anchor links don't seem to work on Docker Hub (https://hub.docker.com/_/php/).

Any idea how this can be handled?

Comment (tianon) [reply php/content.md]: Usually we just reference it in a more general sense as in "see the 'Configuration' section for details".

**Comment by jaylinski** [comment]
@tianon Would be nice if you could give me some feedback. :open_hands: Thx!

**Comment by tianon** [comment]
Doh, sorry, I was waiting for https://github.com/docker-library/official-images/pull/4815 to merge and forgot to come back to it. :+1:

This looks great, thanks!

**Pull request merged**